tests/system/apmserver.py

```diff
@@ -59,7 +60,7 @@ def get_elasticsearch_url(user="", password=""):
         host = os.getenv("ES_HOST", "localhost")
 
         if not user:
-            user = os.getenv("ES_USER", "elastic")
+            user = os.getenv("ES_USER", "admin")
```

Comment:
do we rely on this fallback?

Reply:
No, but in the APM Server setup as well as in the Integration testing we create a superuser `admin` and not `elastic`. For local testing, it is therefore more convenient to use this, as we don't have to explicitly set the env variable then.